system/timing_violation_monitor/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>timing_violation_monitor_utils</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="y-okumura@todo.todo">y-okumura</maintainer>
+  <license>Apache-2.0</license>
```

Comment:
The license is written as `<license>Apache License 2.0</license>` in the other packages, so it might be nicer to align them.

Reply:
The license was aligned with another packages at [b4e9f39](https://github.com/autowarefoundation/autoware.universe/pull/2638/commits/b4e9f3992aa61210fe65afc9ece6225eee73317c)